drivers/base/regmap/regmap-sdw-mbq.c

```diff
@@ -98,4 +98,4 @@ struct regmap *__devm_regmap_init_sdw_mbq(struct sdw_slave *sdw,
 EXPORT_SYMBOL_GPL(__devm_regmap_init_sdw_mbq);
 
 MODULE_DESCRIPTION("Regmap SoundWire MBQ Module");
-MODULE_LICENSE("GPL v2");
+MODULE_LICENSE("GPL");
```

Comment:
this is intended in this commit, right?

Reply:
yes, "GPL v2" is the same as "GPL" and does not describe the actual license details, see https://www.kernel.org/doc/html/latest/process/license-rules.html


“GPL” | Module is licensed under GPL version 2. This does not express any distinction between GPL-2.0-only or GPL-2.0-or-later. The exact license information can only be determined via the license information in the corresponding source files.
-- | --
“GPL v2” | Same as “GPL”. It exists for historic reasons.

So for all new submissions we should use GPL.

Reply:
@plbossart oh, interesting, good to know, thanks! And it's interesting, that they now make no distinction between "only" and "or later" *at the module level*, strange